src/Sylius/Bundle/OrderBundle/Resources/config/doctrine/model/Adjustment.orm.xml

```diff
@@ -45,6 +45,10 @@
         <many-to-one field="orderItem" target-entity="Sylius\Component\Order\Model\OrderItemInterface" inversed-by="adjustments">
             <join-column name="order_item_id" referenced-column-name="id" nullable="true" on-delete="CASCADE"/>
         </many-to-one>
```

Comment:
I think we should discuss it, but in the first step we don't want to do too many changes so they will stay in this PR. We perhaps remove them in second step. I'd like to keep them for the sake of simpler use-cases where applying adjustments at order item level is just fine. I can imagine something like this:

OrderTaxesApplier uses TaxesStrategyInterface

UnitLevelTaxesStrategy (used by default in Sylius)
ItemLevelTaxesStrategy

Anyway, let's do small, quick steps.